api/controllers/MobileAppController.js

```diff
@@ -12,26 +26,56 @@ module.exports = {
       iTunesItemIdentifier: '1002185140'
     }
     */
+    var SUGGEST = 'suggest'
+    var FORCE = 'force'
+    var IOS = 'ios'
+    var ANDROID = 'android'
+    var result = {}
 
-    switch (req.headers['ios-version']) {
-      case '1.6':
-      case '1.65':
-        result = {
-          type: 'force',  // can be 'force' or 'suggest'
-          title: 'A new version of the app is available',
-          message: 'The version you are using is no longer compatible with the site. Please go to the App Store now to update',
-          iTunesItemIdentifier: '1002185140'
-        }
+    switch (req.param('ios-version')) {
+      case '2.0':
+        result = resultBuilder(SUGGEST, IOS)
+        break
+      case '1.9':
+        result = resultBuilder(FORCE, IOS)
         break
       case undefined:
         break
+      default:
+        result = resultBuilder(FORCE, IOS)
     }
 
-    switch (req.headers['android-version']) {
+    switch (req.param('android-version')) {
+      case '2.0':
+        result = resultBuilder(SUGGEST, ANDROID)
+        break
+      case '1.9':
+        result = resultBuilder(FORCE, ANDROID)
+        break
       case undefined:
         break
+      default:
+        result = resultBuilder(FORCE, ANDROID)
     }
 
-    res.ok(result)
+    return res.ok(result)
+  }
+}
+
+function resultBuilder (type, platform) {
+  var iTunesItemIdentifier = '1002185140'
+  var appStoreLink = `https://itunes.apple.com/app/id${iTunesItemIdentifier}`
+  var playStoreLink = 'https://play.google.com/store/apps/details?id=com.hylo.reactnative'
```

Comment:
please have these two urls be environment variables: IOS_APP_STORE_URL and ANDROID_APP_STORE_URL